tests/system/ak_test.go

```diff
@@ -90,12 +90,20 @@ func TestSuite(t *testing.T) {
 }
 
 func setUpSuite(t *testing.T) string {
-	akPath := buildAKBinary(t)
-
 	// https://docs.temporal.io/dev-guide/go/debugging
 	t.Setenv("TEMPORAL_DEBUG", "true")
 
-	return akPath
+	wd, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("failed to get current working directory: %v", err)
+	}
+
+	path, err := filepath.Abs(filepath.Join(wd, "..", "..", "bin", "ak"))
+	if err != nil {
+		t.Fatalf("failed to construct ak binary file path: %v", err)
```

Comment:
failed to find?

Reply:
no, just construct the abs path (it should never fail, but theoretically it can) - but that's a good point, I should have added `Stat()` to check that it is found.